Directory.Build.props

```diff
@@ -3,7 +3,7 @@
   <Import Condition="'$(EnvironmentBuildPropsImported)' != 'True'" Project="$(MSBuildThisFileDirectory)eng\Environment.Build.props" />
 
   <PropertyGroup> 
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
I wouldn't make this change. Let's address any warnings instead.

Reply:
I know. I will not submit that to PR.

It is not that trivial. Namely `main` (net8.0) is still on Gilde 4.15.1. Latest updates were for `net9.0` (RC2).

I could have made bindings for 4.15.1, but then it would be considered "downgrade" and Pobst does not let me do
downgrades...

So with

```xml
  <ItemGroup Condition="$(TargetFramework.Contains('-android'))">
    <PackageReference Include="Xamarin.Android.Glide.AVIF.Integration" Version="4.16.0" />
  </ItemGroup>
```

in the 
    
    `./src/Core/src/Core.csproj` 
    
there are errors:

```shell
./src/Controls/src/Xaml/Controls.Xaml.csproj : error NU1605: Warning As Error: Detected package downgrade: Xamarin.Android.Glide from 4.16.0.7 to 4.15.1.2. Reference the package directly from the project to select a different version.  [./Microsoft.Maui.BuildTasks.slnf]
./src/Controls/src/Xaml/Controls.Xaml.csproj : error NU1605:  Microsoft.Maui.Controls.Xaml -> Microsoft.Maui.Core -> Xamarin.Android.Glide.AVIF.Integration 4.16.0 -> Xamarin.Android.Glide (>= 4.16.0.7)  [./Microsoft.Maui.BuildTasks.slnf]
./src/Controls/src/Xaml/Controls.Xaml.csproj : error NU1605:  Microsoft.Maui.Controls.Xaml -> Microsoft.Maui.Core -> Xamarin.Android.Glide (>= 4.15.1.2) [./Microsoft.Maui.BuildTasks.slnf]
./src/Core/src/Core.csproj : error NU1605: Warning As Error: Detected package downgrade: Xamarin.Android.Glide from 4.16.0.7 to 4.15.1.2. Reference the package directly from the project to select a different version.  [./Microsoft.Maui.BuildTasks.slnf]
./src/Core/src/Core.csproj : error NU1605:  Microsoft.Maui.Core -> Xamarin.Android.Glide.AVIF.Integration 4.16.0 -> Xamarin.Android.Glide (>= 4.16.0.7)  [./Microsoft.Maui.BuildTasks.slnf]
./src/Core/src/Core.csproj : error NU1605:  Microsoft.Maui.Core -> Xamarin.Android.Glide (>= 4.15.1.2) [./Microsoft.Maui.BuildTasks.slnf]
./src/Controls/src/Core/Controls.Core.csproj : error NU1605: Warning As Error: Detected package downgrade: Xamarin.Android.Glide from 4.16.0.7 to 4.15.1.2. Reference the package directly from the project to select a different version.  [./Microsoft.Maui.BuildTasks.slnf]
./src/Controls/src/Core/Controls.Core.csproj : error NU1605:  Microsoft.Maui.Controls.Core -> Microsoft.Maui.Core -> Xamarin.Android.Glide.AVIF.Integration 4.16.0 -> Xamarin.Android.Glide (>= 4.16.0.7)  [./Microsoft.Maui.BuildTasks.slnf]
./src/Controls/src/Core/Controls.Core.csproj : error NU1605:  Microsoft.Maui.Controls.Core -> Microsoft.Maui.Core -> Xamarin.Android.Glide (>= 4.15.1.2) [./Microsoft.Maui.BuildTasks.slnf]
```